docs/hardware-requirements.md

```diff
@@ -0,0 +1,64 @@
+# What hardware requirements are there?
```

Comment:
I think this just needs to be, different hardware lets you run different size models. Generally the bigger the model the higher quality. And then have two tables. one for CPU inference and one for GPU inference. The content on the disk size is good.

Training of models can vary a ton based on the config settings you are using. I think at the very least we could just say generally if you can run inference on it, you can train on it. I was going to say a QLora or Lora but they don't know what that is.

Reply:
Updated to align with this.